floor/test/integration/inheritance/entity_inheritance_test.dart

```diff
@@ -0,0 +1,104 @@
+import 'dart:async';
+
+import 'package:floor/floor.dart';
+import 'package:floor_annotation/floor_annotation.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:sqflite/sqflite.dart' as sqflite;
+
+part 'entity_inheritance_test.g.dart';
+
+void main() {
+  group('dao inheritance tests', () {
+    TestDatabase database;
+    CommentDao commentDao;
+
+    setUp(() async {
+      database = await $FloorTestDatabase.inMemoryDatabaseBuilder().build();
+      commentDao = database.commentDao;
+    });
+
+    tearDown(() async {
+      await database.close();
+    });
+
+    test('use generated inherited insert function', () async {
```

Comment:
⚠️ The test's name doesn't match the test

Reply:
whoops